tasks/mount.yml

```diff
@@ -32,11 +32,11 @@
     - mount
     - shm
 
-- name: create /tmp
+- name: create tmp.mount
   become: 'yes'
   template:
     src: etc/systemd/tmp.mount.j2
-    dest: /lib/systemd/system/tmp.mount
```

Comment:
as you can see by the `src`, it was intended to be placed in `/etc`

Reply:
Yes. It the /etc/ is the right location by systemd specification.